database_generation.d

```diff
@@ -394,14 +394,24 @@ class RecordNotFoundException : Exception {
 	If you just give a type, it assumes the relevant index is "id".
 
 +/
-auto find(alias T)(Database db, int id) {
-
-	// FIXME: if T is an index, search by it.
+static auto find(alias T)(Database db, int id) {
+	// FIXME:
 	// if it is unique, return an individual item.
 	// if not, return the array
+	static if (!is(T)) {
+		static const string fieldName = T.stringof;
```

Comment:
probably should check the DBName here too in case it is overridden. I don't remember if i have code that does this elsewhere already but ill prolly edit this to add it regardless.

Reply:
In line 412 we check for DBName in `tableNameFor`

Reply:
yeah that's for table name, but not field name.